apps/uvc/requirements.txt

```diff
@@ -0,0 +1,2 @@
+--extra-index-url https://artifacts.luxonis.com/artifactory/luxonis-python-snapshot-local/
+depthai==2.10.0.0.dev+e8bbeabeed5b6b6ebdfd6f2a71c1b2348b0af44d
```

Comment:
Need to update to the one mentioned by Alex `depthai==2.13.3.0.dev+b9dbc05ea35d19534678d791f8fbf03ec4b087ba`
Otherwise LGTM, let's merge and add a `temporary` check for windows, that it's not working and document/check for libraries that cause issues on linux system: `sudo apt remove uvcdynctrl uvcdynctrl-data`. It will create huge log files otherwise.

Reply:
@szabi-luxonis I would add the check for libraries along with Ubuntu 21.04 fix (with `python3-pyqt5.sip` module) to the `install_dependencies.sh` script, as it should be responsible for preparing the whole system. Updated the version and added a temporary disabled message in the script itself, thanks!

Reply:
Yes, but older users will not run the script again. 
Either way I agree that we need to add to install dependencies

Reply:
@szabi-luxonis PR with update is here - https://github.com/luxonis/depthai-python/pull/463
> older users will not run the script again
I think we should recommend installing it each time some error happens, just to make sure all of the updated packages are there - as it can be run multiple times in a row without issues

Reply:
Yeah, we should 👍🏻 
In this case it's not a hard error, it will just write tens of GBs of log file to the disk.

Reply:
Actually, it was hard in my case, as I ran out of space available on my Jetson when this package was installed 😀  and couldn't run anything until I deleted this log file

Reply:
We can actually keep things the same (to use `install_requitements.py` still) and execute `install_dependencies.(sh|bat)` inside (as a subprocess call). Since it's safe to run this multiple times, there shouldn't be any downside to it. Will address in a different PR